data/json/snippets/home_pictures.json

```
@@ -0,0 +1,149 @@
+      },
+      {
+        "id": "home_picture_2",
+        "text": "A man in a polo shirt and sunglasses stands in front of a fence holding a leafy branch.  The man grins ear to ear as a giraffe head reaches down and messily chews the leaves off of the branch."
+      },
+      {
+        "id": "home_picture_3",
+        "text": "A worn family photo.  A group of people stand in front of pier next to a shimmering lake.  In the rear center of the crowd stands a tall large built man in his 60's, he is flanked by 6 children of various ages all smiling awkwardly for the camera."
+      },
+      {
+        "id": "home_picture_4",
+        "text": "This blurry photo appears to be at some sort of party.  A group of three people strike goofy poses as a Bride and Groom laugh heartily in the background."
+      },
+      {
+        "id": "home_picture_5",
+        "text": "A family photo of a snowy morning. Two young girls bundled up in heavy winter clothing stand in front of a mountain far in the background the girls are cheerfully smilling.  Which mountain it was you couldn't say."
+      },
+      {
+        "id": "home_picture_6",
+        "text": "A photo of a group of young people in costumes.  They are all dressed in various anime character costumes and are striking poses for the camera.  In the background the entrance of the convention center has writ large on it's entrance 'Animaine'."
+      },
+      {
+        "id": "home_picture_7",
+        "text": "A looming storm dominates the space of this photo, dark clouds scattered promising a miserable rain.  Towards the bottom right of the photo stands a bearded man in a raincoat carrying a fishing rod in one hand and a hooked fish in the other he is brightly lit from the camera's flash which reveals his broad grin.  Written in permanant marker on the picture frame 'In and out just in time!'"
+      },
+      {
+        "id": "home_picture_8",
+        "text": "Two adult men stand front and center wearing paintball masks and comically covered all over in small splashes of paint.  Two boys and a girl surround them with masks still on, cheerfully holding their paintball guns in the air in a celebration of victory."
+      },
+      {
+        "id": "home_picture_9",
+        "text": "A smiling young woman wearing a band t-shirt is in the driver seat of a car, an older man sitting in the passenger seat next to her sweating profusely.  Written in colorful sharpie at the bottom of the frame 'Dad teaching me how to drive a manual for my drivers license'."
+      },
+      {
+        "id": "home_picture_10",
+        "text": "Two dogs are perched on a couch.  One appears to be a rottweiler, the other a black corgi mix.  Both are staring hungrily at the chip the photographer is holding just below the camera."
```

Now who broke into my house and transcribed what happens every time I crack open a bag of chips?
```suggestion
        "text": "Two dogs are perched on a couch.  One appears to be a rottweiler; the other is a black corgi mix.  Both are staring hungrily at the chip the photographer is holding just below the camera."
```